Spelling and grammar fixes (#36)
Pycord-Development/guide: docs/Popular-Topics/sharding.mdx

## What is Sharding?

Sharding is the process of breaking up your bot into smaller pieces so that it has a better time handling
events. As [Discord's Documentation](https://discord.com/developers/docs/topics/gateway#sharding) puts
it, sharding is "a method of user-controlled guild sharding which allows for splitting events across
several gateway connections."

Sharding is used only for large bots and takes up additional resources for managing the new processes.
Pycord automatically shards your bot, so the only thing you need to do is use an `AutoShardedClient`
or `AutoShardedBot` class instead of a plain `Client` or `Bot` class. Pretty cool, right?

Just because you _can_ shard your bot doesn't mean you _should_. While Pycord makes it easy to shard
your bot, it isn't necessary to shard unless your bot is large. Sharding a small bot isn't
just useless, but _harmful_, as it uses extra resources and may slow down your bot. Discord
themselves will let you know when your bot needs sharding so you won't need to worry about this until they contact you.

## How Do I Shard in Pycord?

> Sharding in Pycord is straightforward and extremely easy.

Reviewer: This seems redundant.

Just replace your `discord.Client`, `discord.Bot`, `commands.Bot` objects
with `discord.AutoShardedClient`, `discord.AutoShardedBot`, `commands.AutoShardedBot` respectively.

> These new objects have been inherited from the original objects, so all of the functions from those other types should be present. 

Reviewer: ```suggestion
These new objects are inherited from the original objects, so all of the functions from those other types should be present. 
```
